src/resultset/formatters/resultset_replyverbose.c

```diff
@@ -132,6 +144,13 @@ static void _ResultSet_VerboseReplyWithEdge(RedisModuleCtx *ctx, GraphContext *g
 	_ResultSet_VerboseReplyWithProperties(ctx, gc, (GraphEntity *)e);
 }
 
+static void _ResultSet_VerboseReplyWithArray(RedisModuleCtx *ctx, GraphContext *gc,
+											 SIValue array) {
+	char *str = rm_calloc(512, sizeof(char));
+	int bytesWrriten = SIValue_ToString(array, str, 512);
+	RedisModule_ReplyWithStringBuffer(ctx, str, bytesWrriten);
```

Comment:
I think this should probably have the same logic as the replycompact routine rather than converting the array into a string, right?

Reply:
well...it is more complex
1. you are given a predefined buffer size, and you don't really know the length of the input
2. we want to print only the graph entity id which, if there is any in the array